app/styles/warning.scss

```diff
@@ -0,0 +1,29 @@
+.warning {
+  background: #FFFEC2;
+  font-size: 11px;
+  text-align: left;
+  padding: 5px 0 6px 0;
+}
+
+.warning img {
+  display: inline;
+  width: 15px;
+  float: left;
+  margin: 0px 6px 0px 5px;
+}
+
+.warning p {
+  display: inline;
+}
+
+.warning button {
+  float: right;
+  font-weight: 200;
+  color: #6E68C6;
+  text-decoration: underline;
+  background: #FFFEC2;
+  border: hidden;
+  cursor: pointer;
+  padding: 1px;
+  margin: 0 15px 0 5px;
+}
```

Comment:
We use [BEM naming](http://getbem.com/naming/) for the classes. In case you're not familiar with it it's just that instead of using the HTML tags to style we would use `.warning__image .warning__text .warning__close`. It's a bit easier to maintain as we can more easily change the HTML tags or add tags without breaking the design.